docs/evaluation/simulate.rst

```diff
@@ -45,3 +40,181 @@ NumPy or SciPy to evaluate noise, such as ``numpy.random.standard_normal``.
+backgound (which should not be folded through the ARF) or arbitary
+other components::
+
+  >>> fake_pha(data, model=inst_bkg + my_arf(my_rmf(srcmodel)), is_source=False)
+
+.. image:: ../_static/evaluation/fake_pha.png
+
+
+Adding background
+-----------------
+
+Weak spectral features can be hidden in a large (instrumental or
+astrophysical) background, thus it can be important to include
+background in the simulation.
+
+Sample background from a PHA file
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+One way to include background is to sample it from a
+`~sherpa.astro.data.DataPHA` object. To do so, a background need to be
+loaded into the dataset before running the simluation and, if not done
+before, the scale of the background scaling has to be set::
+
+  >>> data.set_background(read_pha('9774.arf'))
+  >>> data.backscal = 9.6e-06
+  >>> fake_pha(data, srcmdl, add_bkgs=True)
+
+The ``fake_pha`` function simulates the source spectrum as above, but
+then it samples from the background PHA. For each bin, it treats the
+background count number as the expected value and performs a Poisson
+draw. The background drawn from the Poisson distribution is than added
+to the simulated source spectrum (the sum of two Poisson distribution
+is a Poisson distribution again). This works best if the background is
+well exposed and has a large number or counts.
+
+Why do we need to set the ``add_bks=True`` argument and do not simply
+use all available backgrounds? The reason for that is it is often
+useful to read in a file with ``data = read_pha('9774_bg.pi')``, which
+might automatically read in the background as well. Using the
+``add_bkgs=True/False`` to switch the background on or off in the simulation
+makes it easy to compare the same simulation with and without a
+background.
+
+Background models
+^^^^^^^^^^^^^^^^^
+
+When the number of counts in the background is low, the above
+procedure amplifies the noise. The experimental background already
+contains noise just from Poisson statistics. Using the observed count
+numbers as background value and then performing a Poisson draw on them
+again gives a higher level of noise than a real observation would
+have. To avoid this problem, a background model may be used instead::
+
+  >>> from sherpa.models.basic import Const1D
+  >>> bkgmdl = Const1D('bmdl')
+  >>> bkgmdl.c0 = 2
+  >>> fake_pha(data, mdl, add_bkgs=True, bkg_models={'1': bkgmdl})
+
+The keys in the ``bkg_models`` dictionary are the identifiers of the
+backgrounds. Above, we loaded a background with the default identifyer
+(which is ``1``).
+
+
+More than one background
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+If more than one background is set up, then the expected backgrounds
+will be averaged before the Poisson draw. For some backgrounds that
+expected value might be the value of the counts in a PHA file,
+while for others the expected value might be calculated from a model.
+
+
+
+Reference/API
```

Comment:
As I mentioned above I've tried not to put reference sections outside of the `index.rst` file but

a) I may be lying and you can find examples elsewhere
b) not sure what to do here, but I think I'd prefer it to be at a higher level

Reply:
Looking at the higher level I see this: 
```

Reference/API
=============

This section describes some of the classes used to evaluate the
models, in particular in supporting models evaluated against
:py:class:`~sherpa.astro.data.DataPHA` objects.

.. toctree::
   :maxdepth: 2

   astro_background

```

In other words, the next higher level does NOT contain the API docs themselves, instead it refers to a stub page with the APIT docs for `astro_background`. I don't think that `fake` warrants it's own stub_page; instead, I think it should refer to this section here.

I would argue that instead of moving this into its own stub page (to make it look like the docs for `astro_background`, the sub page for `astrop_models` should be expanded to make it look more like this page (with narrative text and the API just at the bottom of that.

Reply:
There's definitely room for us to have a more-principled set up in the documentation, as you've mentioned in #1133, but we (aka me as the reviewer) shouldn't hold back documentation improvements here because of that.